### DISCUSSION TITLE
Update password

### DISCUSSION THREAD

**Discussion by applied-mathematician** [fastapi-users/fastapi-users#1117] [Q&A] [answered]
What's the best approach to update a user's password?  I'm using `on_after_forgot_password` to email the user.  While I could POST to `forgot-password` for a token and then POST to `reset-password`, I don't want the user to be sent a "forgot password" email any time they (or an admin) are changing a password.

**Comment by frankie567** [comment, marked as answer]
You can use the PATCH endpoint and pass the `password` property in the payload: https://fastapi-users.github.io/fastapi-users/10.2/usage/routes/#patch-me

**Reply by applied-mathematician** [reply]
Thanks!
